docs/guides/getting-started/installing-cypress.mdx

```diff
@@ -231,7 +231,7 @@ requirements:
 ### Linux Prerequisites
 
 If you're using Linux, you'll want to have the required dependencies installed
-on your system. Depending on your system defaults, these dependencies may already be installed.
+on your system. Cypress on Linux requires a minimum [`glibc`](https://www.gnu.org/software/libc/) version of `2.17`. Depending on your system defaults, these dependencies may already be installed.
```

Comment:
That does not read so well in front of the sentence "Depending on your system defaults, these dependencies may already be installed." as it implies that you can change it, whereas it is a fixed version completely tied to the version of the operating system you are using. It would be better as a separate paragraph.

Reply:
How is something like 091fe973d6866e3041db8930edc3b3f5f6ed297a?

Reply:
I'm fine with the replacement paragraph.